app/js/lib/foxbox/recipes.js

```diff
@@ -135,8 +135,8 @@ export default class Recipes {
   }
 
   getGetters() {
-    // Currently we support only Clock as a trigger.
-    const supportedKinds = ['CurrentTimeOfDay'];
+    // Currently we support only Clock and Motion Sensor as triggers.
+    const supportedKinds = ['CurrentTimeOfDay', 'OpenClosed'];
```

Comment:
For now we consider ```OpenClosed``` as Motion Sensor, but later we should introduce dedicated ChannelKind